[2024-02-25T13:06:21.627+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-08T00:00:00+00:00 [queued]>
[2024-02-25T13:06:21.742+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-08T00:00:00+00:00 [queued]>
[2024-02-25T13:06:21.743+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2024-02-25T13:06:21.946+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): second_task> on 2021-08-08 00:00:00+00:00
[2024-02-25T13:06:21.973+0000] {standard_task_runner.py:57} INFO - Started process 384 to run task
[2024-02-25T13:06:22.038+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-08T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpa0kx6yv4']
[2024-02-25T13:06:22.060+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask second_task
[2024-02-25T13:06:22.374+0000] {task_command.py:410} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-08T00:00:00+00:00 [running]> on host 0c40846c7775
[2024-02-25T13:06:24.312+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v5' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-08-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-08-08T00:00:00+00:00'
[2024-02-25T13:06:24.313+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-25T13:06:24.321+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2024-02-25T13:06:24.364+0000] {subprocess.py:86} INFO - Output:
[2024-02-25T13:06:24.372+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2024-02-25T13:06:24.378+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-02-25T13:06:24.484+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210808T000000, start_date=20240225T130621, end_date=20240225T130624
[2024-02-25T13:06:24.685+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-02-25T13:06:24.810+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
